src/style.css

```diff
@@ -2,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  outline: none;
+  display: block;
+  width: 100%;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search__input::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.search__input:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
It seems like you've made a mistake here. The `background-color` property does not accept `linear-gradient` as a value. You should use `background` or `background-image` property for this.